executor/api/rabbitmq/connection.go

```diff
@@ -107,6 +108,17 @@ func (c *connection) connect() error {
 			c.log.Error(err, "error creating rabbitmq channel", "uri", c.uri)
 			return fmt.Errorf("error '%w' creating rabbitmq channel to %q", err, c.uri)
 		}
+
+		err = c.channel.Qos(
```

Comment:
@ddl-plouie  FYI -- I also made this change to capture the error here, this is a common golang pattern, the error needs to get propagated rather than being ignored